docker/build_docker_oss4local.sh

```diff
@@ -62,7 +69,7 @@ fi
 DOCKER_BUILDKIT=1 docker build \
     --build-arg DEVICE=$DEVICE_TYPE \
     --build-arg SOURCE=${SOURCE_IMAGE} \
-    -f "${GSF_HOME}/docker/Dockerfile.local" . -t $DOCKER_FULLNAME
+    -f "${GSF_HOME}/docker/local/Dockerfile.local" . -t $DOCKER_FULLNAME
```

Comment:
What is the difference between docker/local/Dockerfile.local and docker/Dockerfile.local?

Reply:
Forgot to delete the old version, I'm just moving the local Dockefile to its own directory, so it matches the layout of sagemaker and parmetis that also exist in their own dir.